speaking-portal-project/src/animationFactory/ffmpeg.ts

```diff
@@ -15,6 +15,8 @@ export async function ffmpegProcessor(audio_file: string, text_file: string, out
         `${audio_file}`,
         '-r',
         '24',
+        '-s',
+        '720x480', // added to avoid "width not divisible by 2" error
```

Comment:
I wonder if we should add a parameter for the output video size?

Reply:
I'm not sure if the user would choose the dimension or if it's based on the client. This is a great call out.
@mkuelker can we please add output video size/dimensions as a question for our client in case they have specific dimensions in mind?

Reply:
Yeah, I'll add that to the agenda.

Reply:
One thing worth noting is that if we're using pngs and crazy high resolutions we could get some pixelation. We'll have to run tests to see how much of an effect there is